obspy/io/mseed/util.py

```diff
@@ -238,31 +275,110 @@ def get_timing_and_data_quality(file_or_file_object):
     info = get_record_information(file_or_file_object)
     # Keep track of the extracted information.
     quality_count = [0, 0, 0, 0, 0, 0, 0, 0]
+    activity_count = [0, 0, 0, 0, 0, 0, 0]
+    io_count = [0, 0, 0, 0, 0, 0]
```

Comment:
I've implemented this feature but it is not as straightforward as I thought to keep the old tests running. The method values() is actually returning something like this odict_values([0, 0, 0, 0, 0, 0, 0, 0]) which is a dictionary view object instead of the simple array. I must say I'm using python 3 and this seems to be causing it. I can solve this but I'm not sure how this would behave with python 2